src/Tribe/RSVP.php

```diff
@@ -118,6 +118,12 @@ class Tribe__Tickets__RSVP extends Tribe__Tickets__Tickets {
 	 */
 	public $deleted_product = '_tribe_deleted_product_name';
 
+	/**
+	 * Meta key that holds the "not going" option visibility status.
+	 * @var string
```

Comment:
:no_entry_sign: There must be exactly one blank line before the tags in a doc comment

<sub>hash: dd347afdf19e3a6741749462d984517e</sub>

Reply:
```suggestion
	 *
	 * @var string
```